src/lib/viewers/office/__tests__/OfficeLoader-test.js

```diff
@@ -73,13 +76,26 @@ describe('lib/viewers/office/OfficeLoader', () => {
             });
 
             it('should not return a viewer if the file is a password-protected shared link', () => {
-                const editedFakeFile = fakeFile;
-                editedFakeFile.shared_link = {
-                    is_password_enabled: true,
+                const editedFakeFile = {
+                    ...fakeFile,
+                    shared_link: {
+                        is_password_enabled: true,
+                    },
                 };
                 const viewer = OfficeLoader.determineViewer(editedFakeFile, []);
                 expect(viewer).to.equal(undefined);
             });
+
+            it('should respect maxFileSize in viewerOptions', () => {
+                const viewerOptions = {
+                    office: {
+                        excelOnlineAllowedMaxFileSize: FIVE_MB + 1,
+                    },
+                };
+                const editedFakeFile = { ...fakeFile, size: FIVE_MB + 1 };
+                const viewer = OfficeLoader.determineViewer(editedFakeFile, [], viewerOptions);
```

Comment:
To test that it is respecting the passed in option, should it instead expect the viewer to be returned despite the size being over the default FIVE_MB?